.circleci/make-rpm-packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "PARAM_RELEASE: $PARAM_RELEASE"
+echo "PARAM_DIST: $PARAM_DIST"
+
+if test "x$PARAM_RELEASE" = "x"; then
```

Comment:
can we leave the 70s behind and just use `[ -z "$PARAM_RELEASE" ]` or is there a reason to continue using this pattern?

Reply:
No reason! Changed it!